DHL/Datatype/GB/ExportLineItem.php

```diff
@@ -111,6 +111,16 @@ class ExportLineItem extends Base
             'required' => false,
             'subobject' => false,
         ), 
+        'ManufactureCountryCode' => array(
+            'type' => 'string',
```

Comment:
This line is different from the change in Pull Request #2. This is set as `''` (empty string) in the other PR. It seems that the empty string is correct, based on the other lines in this file.